apps/condo/domains/news/components/NewsForm/UpdateNewsForm.tsx

```diff
@@ -0,0 +1,164 @@
+import dayjs from 'dayjs'
+import get from 'lodash/get'
+import isEmpty from 'lodash/isEmpty'
+import { useRouter } from 'next/router'
+import React, { useCallback, useMemo } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { ActionBar, Button } from '@open-condo/ui'
+
+import LoadingOrErrorPage from '@condo/domains/common/components/containers/LoadingOrErrorPage'
+import { NewsItem, NewsItemScope, NewsItemTemplate } from '@condo/domains/news/utils/clientSchema'
+
+import { BaseNewsForm, BaseNewsItemFormProps } from './BaseNewsForm'
+import { CreateNewsActionBar, getCompletedNotification } from './CreateNewsForm'
+
+export interface IUpdateNewsForm {
+    id: string
+}
+
+export const UpdateNewsActionBar = (props) => {
+    const intl = useIntl()
+    const SaveButtonMessage = intl.formatMessage({ id: 'Save' })
+
+    const { handleSave, isLoading } = props
+
+    return (
+        <ActionBar
+            actions={[
+                <Button
+                    key='submit'
+                    type='primary'
+                    children={SaveButtonMessage}
+                    onClick={handleSave}
+                    disabled={isLoading}
+                />,
+            ]}
+        />
+    )
+}
+
+
+export const UpdateNewsForm: React.FC<IUpdateNewsForm> = ({ id }) => {
+    const intl = useIntl()
+    const ServerErrorMessage = intl.formatMessage({ id: 'ServerError' })
+    const EmptyTemplateTitle = intl.formatMessage({ id: 'news.fields.emptyTemplate.title' })
+    
+    const router = useRouter()
+
+    const {
+        loading: newsItemLoading,
+        obj: newsItem,
+        error: newsItemError,
+    } = NewsItem.useObject({
+        where: { id },
+    })
+
+    const {
+        allDataLoaded: newsItemScopeAllDataLoaded,
+        objs: newsItemScopes,
+        error: newsItemScopeError,
+    } = NewsItemScope.useAllObjects({
+        where: {
+            newsItem: { 
+                id: id,
+            },
+        },
+    })
+
+    const organizationId = useMemo(() => get(newsItem, 'organization.id', null), [newsItem])
+
+    const {
+        loading: isNewsFetching,
+        objs: allNews,
```

Comment:
when I get allNews, I mean all news items of the organisation, I need it to know if the user wants to send more than 5 news items in a day to show a warning in that case

should I add an extra WHERE, where I get newsItems to get only newsItems that have been sent today or are scheduled to be sent today or in the future? I don't really see why this might be better

Reply:
It's better to use `useCount()` with `where` filter by date.